Sources/PointFree/Account/Account.swift

```diff
@@ -772,7 +773,7 @@ private func mainAction(for subscription: Stripe.Subscription) -> Node {
           action(path(to: .account(.subscription(.change(.update(nil)))))),
           method(.post),
           onsubmit(unsafe: """
-if (!confirm("Upgrade to yearly billing? You will be charged \(formattedAmount ?? "") immediately with a pro-rated refund for the time remaining in your billing period.")) {
+if (!confirm("Upgrade to yearly billing? You will be charged \(formattedAmount ?? "") immediately with a prorated refund for the time remaining in your billing period.")) {
```

Comment:
I think it's more correct to use "prorated" over "pro-rated". is that true?

Reply:
ah yup